_pages/getting-started/getting-started.md

```diff
@@ -148,7 +148,7 @@ If you're looking to go into greater detail,
 but it can also be a little overwhelming.
 If you ever need a quick answer, clarification or just general Security and Compliance help, fill out the [TTS Tech Portfolio General Request Form](https://forms.gle/wPrCXJqDgaksXRwV8) and selecting the option for a _New System (ATO/Architecture consult)_.
 
-If you hear "ATO" or "Authority to Operate" and are wondering what that's about,
+If you hear "ATO" or "Authority to Operate" for TTS custom developed software and are wondering what that's about,
```

Comment:
I think the clarification raises more questions than it answers. Like "what about other software" and "how do I know"

The person reading this doesn't even know what the letters ATO mean. They're not in a position yet to get the nuances. 

I think a better approach would be to leave BYS as the definitive ATO resource (including defining what an ATO is and what software it's for) and leave this as "if you have questions, go here..."

Reply:
I updated the [issue](https://github.com/18F/tts-tech-portfolio/issues/291) to update BYS with this comment, and will close this out in favor of handling it there.